build/webcharts.js

```diff
@@ -1951,6 +1957,25 @@ function drawArea(area_drawer, area_data, datum_accessor, class_match, bind_acce
   return area_grps;
 }
 
+function destroy() {
+  var destroyControls = arguments.length <= 0 || arguments[0] === undefined ? true : arguments[0];
+
+  //run onDestroy callback
+  this.events.onDestroy.call(this);
+
+  //remove resize event listener
+  var context = this;
+  d3.select(window).on('resize.' + context.element + context.id, null);
+
+  //destroy controls
+  if (destroyControls && this.controls) {
```

Comment:
You can bind a set of controls to multiple charts, so I was mainly thinking of a use case where you wanted to destroy one of several charts using the same controls.